### DISCUSSION TITLE
Only subtitles from immersive on youtube

### DISCUSSION THREAD

**Discussion by Root-FTW** [immersive-translate/immersive-translate#1018] [Ideas]
### 新功能 | New Feature

With the new CC translation function in Youtbe it is very annoying to see double subtitles in the videos the original and the translated one, is it possible to see only the translated one by immersive?

### 为什么您想要添加这些功能 | Why you want to add these features

1.
2.
3.

**Comment by theowenyoung** [comment]
But if you only want to see translated subtitles, you can use the original youtube subtitle?

Immersive Translate dual subtitle is for those want the dual subtitles people...

**Reply by Root-FTW** [reply]
I only speak Spanish, I see a lot of videos in English where there are only the CC in English, with the latest update of Immersive now I see the CC in English + translated to Spanish, which is GREAT, now I can understand what they say in the video since Immersive now shows them in Spanish.

I am a paid PRO user of Immersive so I use Deepl's technology to see things in my language.

It would be great if on Youtube I could only see the subtitles in my language, since seeing double subtitles (Original + My language) is a lot of text that distracts from the video.

**Reply by theowenyoung** [reply]
Thanks, I got you, we are planning to support this if you change to [translation only mode]
